app/test_main.py

```diff
@@ -1,3 +1,29 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_ages():
+    ages = {"cat_age": 14, "dog_age": 14, "human_age": [0, 0]}
+    assert get_human_age(
+        ages["cat_age"], ages["dog_age"]) == ages["human_age"]
+
+
+def test_first_ages():
+    ages = {"cat_age": 15, "dog_age": 15, "human_age": [1, 1]}
+    assert get_human_age(
+        ages["cat_age"], ages["dog_age"]) == ages["human_age"]
```

Comment:
```suggestion
    assert get_human_age(15, 15) == [1, 1]
```
It's better to use explicit values, it makes the code more readable